taxonomies/categories.txt

```diff
@@ -44760,12 +46302,15 @@ nl:Gedroogde groenten
 
 <en:Dried vegetables
 <en:Tomatoes and their products
-en:Dried tomatoes
+en:Dried tomato, Dried tomatoes, Sun-Dried tomato, Sun-Dried tomatoes
```

Comment:
for categories, by convention the plural should always come first : "Dried tomatoes", it will be the canonical name of the category.

Reply:
Je ne connaissais pas la convention "Pluriel en premier", merci pour l'info ! Corrigé dans la prochaine PR.